### ISSUE TITLE
CodePath Prework Submission for Android class

### ISSUE THREAD

**Issue by oscarso** [oscarso/SimpleTodo#1]
@codepathreview and @codepath
Please review my pre-work.
GIF Walkthrough:
https://github.com/oscarso/SimpleTodo/blob/master/SimpleTodo_GIF_walkthrough.gif

Thank you!

Best,
Oscar

**Comment by nesquena** [comment]
Oscar, 

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project-based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer to learn all practical Android development and best practices as quickly as possible. We also do a code review for each submitted project once the program begins.

Since there have been over 60 applications and we can only accept 15 people into the pilot course, the next step is to **work on one or two [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo)**. This course is project-based and the optional extensions on each project are often the most valuable learnings since they dive deeper into common Android use cases.

Once you submit 1-2 extensions, your application will be passed along to the organizers for final enrollment. Let me know if you have any questions in the meantime.

**Comment by oscarso** [comment]
Thanks Nathan.I am out of town on vacation from Nov 14 - 23. But, I will try my best to add more extensions.I have already published this "SimpleTodo" app:https://play.google.com/store/apps/details?id=simpletodo.oscarsoft.co.simpletodo
Will the effort of publishing this app be considered as well ?
Best,Oscar

Date: Sun, 15 Nov 2015 10:49:05 -0800

**Comment by nesquena** [comment]
Yes, that will be considered. Will discuss with the organizers as well. Have a great vacation.

**Comment by oscarso** [comment]
I am working on the SimpleTodo app now on the Amtrak train; adding the SQLite extension.
I also have another app called updish under my name at the store.
Moreover, I have an iPhone app called ShakerBet at Apple Store as well.

Best,
Oscar

**Comment by nesquena** [comment]
Great. Will keep an eye out for your extension as well.

**Comment by oscarso** [comment]
I just added the SQLite support, and pushed it. Although the solution is NOT the best. I am running out of time... if I have to submit before midnight of Nov 20th.  The train that I am on right now is moving away from the cell network coverage area...
Also, updated GIF Walkthrough:
https://github.com/oscarso/SimpleTodo/blob/master/SimpleTodo_GIF_walkthrough.gif

**Comment by nidhi1608** [comment]
Great to see you were able to augment your app with SQLite support. As mentioned earlier, the extension tasks available on each project are often the most valuable learnings since they dive deeper into common real-world Android use cases.

Your application will be passed along to the organizers for final admissions. You should hear back again soon. Let me know if you have any questions in the meantime.

**Comment by oscarso** [comment]
Thanks.If there is anything you want me to add in order to secure the slot, please let me know.I am back from the Amtrak trip.
Best,Oscar

Date: Mon, 23 Nov 2015 11:30:51 -0800

**Comment by nidhi1608** [comment]
It would help if you can submit any more extensions by tonight. The organizers are going to finalize the list of participants based on the submission, so anything that you can add will be helpful.

**Comment by oscarso** [comment]
I see, thanks.I have also published the app:https://play.google.com/store/apps/details?id=simpletodo.oscarsoft.co.simpletodo
Best,Oscar

Date: Tue, 24 Nov 2015 18:35:48 -0800
